src/apps/virtio_net/virtio_net.lua

```diff
@@ -0,0 +1,76 @@
+-- Application to connect to a virtio-net driver implementation
+--
+-- Licensed under the Apache 2.0 license
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Copyright (c) 2015 Virtual Open Systems
```

Comment:
I was about to weigh in with support for avoiding these headers, but for folks like vosys and igalia that do work for multiple customers, potentially assigning some copyright to external entities, it's probably important to write down who owns that copyright.  I don't know of a nice way to avoid copyright headers in this regard.